build.js

```diff
@@ -0,0 +1,21 @@
+const rollup = require('rollup');
+const babel = require('rollup-plugin-babel');
+
+rollup.rollup({
+    entry: './index.js',
+    plugins: [
+        babel({
+            exclude: 'node_modules/**',
+        }),
+    ],
+}).then((bundle) => {
+    bundle.write({
+        format: 'umd',
+        moduleId: 'jquery-csrf-token',
+        moduleName: 'jqueryCSRFToken',
```

Comment:
ok, I actually like the accronyms to be uppercase but I don't want to bikeshed it too much